packages/utils/src/period/chartGranularities.js

```diff
@@ -1,15 +1,14 @@
 /**
  * Tupaia Web
```

Comment:
Can we please name this file `periodGranularities.js` (since this will end up being the source of truth for all that stuff eventually) and rename the existing `periodGranularities` file to `getDateRangeForGranularity` (since that is all the file exports)